VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -257,10 +258,13 @@ export function HomeScreen({}: HomeScreenProps) {
             </Box>
           ) : (
             <Box gap={theme.dimensions.condensedMarginBetween} mx={theme.dimensions.condensedMarginBetween}>
-              {!!upcomingAppointmentsCount && (
+              {!!upcomingAppointmentsCount && upcomingDaysLimit && (
```

Comment:
```suggestion
              {!!upcomingAppointmentsCount && !!upcomingDaysLimit && (
```
This shouldn't ever happen, but if the `upcomingDaysLimit` was returned as `0`, which is also the default value in the Redux store, this would prevent an error from being thrown